src/cml.js

```diff
@@ -335,6 +335,8 @@ class CML {
       userEmail = GIT_USER_EMAIL,
       userName = GIT_USER_NAME
     } = opts;
+    await exec('git config --global --add safe.directory "$PWD"');
+    await exec("git config --global --add safe.directory '*'");
```

Comment:
both `$PWD` and `*`?

Reply:
Yes; `git < 2.36, > 2.35.1` and `git >= 2.36` respectively

Reply:
Still, I'm preparing a better solution. I plan to push one more commit before merging.

Reply:
does `$PWD` not work on all versions?

Reply:
Yes, it does, but it breaks if you aren't in the repository root